pages/index.tsx

```diff
@@ -1,10 +1,46 @@
-import styled from 'styled-components'
+import React from "react";
+import styled from "styled-components";
+import IntroSection from "../components/intro-section/intro-section";
+import ProfileSection from "../components/profile-section/profile-section";
+import TrustedBySection from "../components/trusted-section/trusted-section";
 
-const Title = styled.h1`
-  color: red;
-  font-size: 50px;
-`
+const MainLayout = styled.div`
+  padding: 2rem 6rem;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-column-gap: 4rem;
+  grid-row-gap: 1rem;
+
+  .profile {
+    margin-top: 1rem;
+  }
+  
+  @media only screen and (max-width: 600px) {
+    padding: 0;
+  }
+
+  @media only screen and (min-width: 992px) {
+    grid-template-columns: 1fr 3fr;
+
+    .profile {
+      justify-self: center;
+    }
+  }
+`;
+
+const RightPanelStyle = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
```

Comment:
You can delete flex here because the children's are div's and the default behaviour is a column layout.